<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SaleDialog</class>
 <widget class="QDialog" name="SaleDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>610</width>
    <height>250</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>610</width>
    <height>250</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>610</width>
    <height>250</height>
   </size>
  </property>
  <property name="font">
   <font>
    <family>Comic Sans MS</family>
   </font>
  </property>
  <property name="windowTitle">
   <string>Продати товар</string>
  </property>
  <property name="windowIcon">
   <iconset resource="pics.qrc">
    <normaloff>:/pics/sale_goods.png</normaloff>:/pics/sale_goods.png</iconset>
  </property>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>291</width>
     <height>231</height>
    </rect>
   </property>
   <layout class="QFormLayout" name="formLayout">
    <property name="verticalSpacing">
     <number>8</number>
    </property>
    <item row="0" column="0">
     <widget class="QLabel" name="label">
      <property name="text">
       <string>Бренд</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QComboBox" name="brand_cb"/>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>Модель</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QComboBox" name="model_cb"/>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="label_6">
      <property name="text">
       <string>Розмір</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QComboBox" name="size_cb"/>
    </item>
    <item row="3" column="0">
     <widget class="QLabel" name="label_3">
      <property name="text">
       <string>Ціна продажу, грн</string>
      </property>
     </widget>
    </item>
    <item row="3" column="1">
     <widget class="QDoubleSpinBox" name="price_sb">
      <property name="maximum">
       <double>9999999.000000000000000</double>
      </property>
      <property name="singleStep">
       <double>10.000000000000000</double>
      </property>
     </widget>
    </item>
    <item row="4" column="0">
     <widget class="QLabel" name="label_5">
      <property name="text">
       <string>Кількість</string>
      </property>
     </widget>
    </item>
    <item row="4" column="1">
     <widget class="QSpinBox" name="count_sb">
      <property name="minimum">
       <number>1</number>
      </property>
     </widget>
    </item>
    <item row="6" column="0">
     <widget class="QLabel" name="label_7">
      <property name="text">
       <string>Ціна оптова, грн</string>
      </property>
     </widget>
    </item>
    <item row="6" column="1">
     <widget class="QDoubleSpinBox" name="wholesale_price_sb">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>22</height>
       </size>
      </property>
      <property name="font">
       <font/>
      </property>
      <property name="styleSheet">
       <string notr="true">QDoubleSpinBox {
background-color: rgba(250,250,250,0.2);
}</string>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
      <property name="buttonSymbols">
       <enum>QAbstractSpinBox::NoButtons</enum>
      </property>
      <property name="maximum">
       <double>9999999.000000000000000</double>
      </property>
     </widget>
    </item>
    <item row="7" column="0">
     <widget class="QLabel" name="lbl">
      <property name="text">
       <string>Ціна роздрібна, грн</string>
      </property>
     </widget>
    </item>
    <item row="7" column="1">
     <widget class="QDoubleSpinBox" name="retail_price_sb">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>22</height>
       </size>
      </property>
      <property name="font">
       <font/>
      </property>
      <property name="styleSheet">
       <string notr="true">QDoubleSpinBox {
background-color: rgba(250,250,250,0.2);
}</string>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
      <property name="buttonSymbols">
       <enum>QAbstractSpinBox::NoButtons</enum>
      </property>
      <property name="maximum">
       <double>9999999.000000000000000</double>
      </property>
     </widget>
    </item>
    <item row="5" column="0" colspan="2">
     <spacer name="verticalSpacer">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>40</height>
       </size>
      </property>
     </spacer>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>10</y>
     <width>269</width>
     <height>231</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QLabel" name="pic_lbl">
      <property name="minimumSize">
       <size>
        <width>267</width>
        <height>150</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>267</width>
        <height>150</height>
       </size>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap resource="pics.qrc">:/pics/default_pic.png</pixmap>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item>
     <spacer name="verticalSpacer_2">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>40</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="ok_pb">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="text">
         <string>ОК</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="cancel_pb">
        <property name="text">
         <string>Скасувати</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources>
  <include location="pics.qrc"/>
 </resources>
 <connections/>
</ui>
